strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/storage/repository/RepositoryPolicyEnum.java

```diff
@@ -1,14 +1,30 @@
 package org.carlspring.strongbox.storage.repository;
 
+import java.util.stream.Stream;
+
 /**
  * @author mtodorov
  */
 public enum RepositoryPolicyEnum
 {
 
-    RELEASE("release"),
+    RELEASE("release")
+            {
+                @Override
+                public boolean acceptsSnapshots()
+                {
+                    return false;
+                }
+            },
 
-    SNAPSHOT("snapshot"),
+    SNAPSHOT("snapshot")
+            {
+                @Override
+                public boolean acceptsReleases()
+                {
+                    return false;
+                }
+            },
 
     MIXED("mixed");
```

Comment:
If you were to do things through this `enum`, then you would have to also implement these two methods for `MIXED` and return true for them, right...?